exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java

```diff
@@ -169,4 +176,223 @@ private static boolean containIdentity(List<? extends RexNode> exps,
     }
     return true;
   }
+
+  /**
+   * Copied from {@link RelOptUtil#splitJoinCondition(RelNode, RelNode, RexNode, List, List)}. Modified to rewrite
```

Comment:
Is there a way to make this not copied? @julianhyde, any ideas for how to refactor common stuff out?